bundles/GraphicServiceOrder/Service/OrderService.php

```diff
@@ -28,17 +29,19 @@ class OrderService
     private $entityManager;
     private $messageBus;
     private $ownCloudFilesFolder;
+    private $tokenStorage;
 
     /**
      * OrderService constructor.
      *
-     * @param \Doctrine\ORM\EntityManagerInterface              $entityManager
-     * @param \App\Service\HammerService                        $hammerService
-     * @param \App\Service\OwnCloudService                      $ownCloudService
-     * @param \GraphicServiceOrder\Repository\GsOrderRepository $gsOrderRepository
-     * @param \Symfony\Component\HttpKernel\KernelInterface     $appKernel
-     * @param \Symfony\Component\Messenger\MessageBusInterface  $messageBus
-     * @param string                                            $ownCloudFilesFolder
+     * @param \Doctrine\ORM\EntityManagerInterface                                                $entityManager
```

Comment:
Why newlines before parameter names?

Reply:
That was done as part of an earlier code review.